paper-scroll-header-panel.html

```diff
@@ -384,6 +427,25 @@
       }
     },
 
+    _querySelectorInHeader: function(selector) {
+      return Polymer.dom(this.header).querySelector(selector);
+    },
+
+    _prepareHeaderTitle: function(headerTitle, oldHeaderTitle) {
+      var headerTitleElement;
+
+      if (headerTitle) {
+        if (headerTitleElement = this._querySelectorInHeader(headerTitle)) {
+          headerTitleElement.classList.add('header-title');
+        }
+      }
+      if (oldHeaderTitle) {
+        if (headerTitleElement = this._querySelectorInHeader(oldHeaderTitle)) {
+          headerTitleElement.classList.remove('header-title');
+        }
+      }
+    },
+
```

Comment:
We can remove `_prepareHeaderTitle` if we don't make the `headerTitleSelector` customizable.